### PR TITLE
feat(time-picker): ✨  add keyboard navigation to the column

### PR DESCRIPTION
**Pull request by navin-moorthy**
Adds keyboard arroa key navigation to Timepicker Column

### CODE REVIEW

File: src/timepicker/TimePickerContent.ts [hunk 1]
```diff
@@ -21,6 +24,40 @@ export const useTimePickerContent = createHook<
...
+  useProps(_, { ref: htmlRef, onKeyDown: htmlOnKeyDown, ...htmlProps }) {
+    const ref = React.useRef<HTMLElement>(null);
+
+    const onKeyDown = React.useMemo(() => {
+      return createOnKeyDown({
+        onKeyDown: htmlOnKeyDown,
+        keyMap: () => {
+          return {
+            ArrowRight: () => {
+              if (ref.current) {
```
Comment (anuraghazra) [inline src/timepicker/TimePickerContent.ts:37]: I think we can put the `if (ref.current)` check outside ArrowRight & ArrowLeft. before the return

Comment (anuraghazra) [reply src/timepicker/TimePickerContent.ts:37]: Throws TS null assertion error.

Comment (navin-moorthy) [reply src/timepicker/TimePickerContent.ts]: Added Guard Clause instead as discussed.

**Review by anuraghazra — APPROVED** [review]

**Comment by anuraghazra** [comment]
LGTM! 🎉

**Pull request merged**